packages/story-editor/src/components/panels/design/border/borderWidth.js

```diff
@@ -87,24 +56,25 @@ const ToggleWrapper = styled.div`
   align-self: stretch;
 `;
 
-function UnLockedInput({ labelText, ...rest }) {
-  return (
-    <>
-      <Separator />
-      <Label>
-        <NumericInput
-          containerStyleOverride={inputContainerStyleOverride}
-          {...rest}
-        />
-        <LabelText>{labelText}</LabelText>
-      </Label>
-    </>
-  );
-}
+const StackableInput = styled(NumericInput)`
+  & > div {
+    border-radius: 0;
+    padding: 2px 7px 2px 12px;
+    ${inputContainerStyleOverride};
+  }
+`;
```

Comment:
I feel like this should be a generalized component as well as its the same between the two panels?